src/resources/ui/xclicker-window.ui

```diff
@@ -558,14 +553,16 @@
                         <property name="max-length">6</property>
                         <property name="width-chars">0</property>
                         <property name="placeholder-text" translatable="yes">ms</property>
-                        <signal name="insert-text" handler="insert_handler" swapped="no" />
+                        <signal name="insert-text" handler="insert_handler" swapped="no"/>
                       </object>
                       <packing>
                         <property name="left-attach">1</property>
                         <property name="top-attach">3</property>
                       </packing>
                     </child>
-
+                    <child>
```

Comment:
Whats this?

Reply:
I haven't touched ui file by hands. Glade does this..

Reply:
I guess, it will be the same for everyone who edit ui files in Glade.